docs/contributor/releasing.md

```diff
@@ -59,7 +65,7 @@ This release process covers the steps to release new major and minor versions fo
      git push upstream {RELEASE_VERSION}
      ```
 
-10. If the previous release was a bugfix version (patch release) that contains cherry-picked changes, these changes might appear again in the generated change log. Edit the release description and remove redundant entries if necessary.
+11. If the previous release was a bugfix version (patch release) that contains cherry-picked changes, these changes might appear again in the generated change log. Edit the release description and remove redundant entries if necessary.
```

Comment:
I think it can also mean changes that appear there and they shouldn't, not that they are necessarily redundant. Or maybe things that are missing from the auto-generated description, and thus should be added.

Reply:
@TeodorSAP Just to let you know, there should be no missing changes in the auto-generated change logs. It is only the case that if the previous release was a patch release that contains cherry-picked changes, then these changes will appear again in the current release and we should remove them

Reply:
Ok, then I will implement Nina's review